rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -803,6 +807,196 @@ private void assertEventWasEmittedWithExpectedData(byte[] expectedData) {
                 .findFirst();
             assertTrue(data.isPresent());
         }
+
+        @Test
+        void registerBtcTransaction_forSvpFundTransaction_whenProposedFederationDoesNotExist_shouldRegisterTransactionButNotUpdateSvpFundTransactionValues() throws Exception {
+            // arrange
+            BtcTransaction svpFundTransaction = createAndSaveSvpFundTransactionUnsigned();
+            signInputs(svpFundTransaction); // a transaction trying to be registered should be signed
+            setUpForTransactionRegistration(svpFundTransaction);
+
+            when(federationSupport.getProposedFederation()).thenReturn(Optional.empty());
+
+            // act
+            int activeFederationUtxosSizeBeforeRegisteringTx = federationSupport.getActiveFederationBtcUTXOs().size();
+            bridgeSupport.registerBtcTransaction(rskTx, svpFundTransaction.bitcoinSerialize(), btcBlockWithPmtHeight, pmtWithTransactions.bitcoinSerialize());
+            bridgeStorageProvider.save();
+
+            // assert
+            assertNewUtxoWasSaved(activeFederationUtxosSizeBeforeRegisteringTx);
```

Comment:
What would happen if the `fundTx` would not contain a change output to the current federation? I am assuming nothing would happen and normal flow should continue, but just double checking. 

Maybe what I am trying to say is, does it really matter that we check if a new UTXO was registered due to the `fundTx` being registered?

Reply:
is a very good question, im adding a test without the change output. Thanks!